.github/oauth-test/compose.yaml

```diff
@@ -66,6 +70,8 @@ services:
       - SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_TYPE=jwt
     ports:
       - '8081:8081'
+    volumes:
```

Comment:
I preferred this solution because these are data that in some instances may not be needed. If we include it in the docker image those data will always be loaded. For example, the data source https://bbmritestnn.gcc.rug.nl may be probably removed in production.

Reply:
yes, but I would include it in the docker file but make it loaded only for profile dev. Otherwise the setup process becomes more difficult.